specification/synapse/data-plane/Microsoft.Synapse/preview/2021-07-01-preview/symsSync.json

```diff
@@ -0,0 +1,268 @@
+        "responses": {
+          "201": {
+            "description": "Request Accepted Response.",
+            "schema": {
+              "$ref": "#/definitions/acceptedResponse"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/workspace.json#/definitions/ErrorContract"
+            }
+          }
+        }
+      }
+    },
+    "/symssync/update": {
+      "put": {
+        "tags": [
+          "symssync"
+        ],
+        "operationId": "SymsSync_Update",
+        "description": "Update files in Syms",
+        "x-ms-examples": {
+          "Register files in Syms": {
+            "$ref": "./examples/SymsSync_Update.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "body",
+            "name": "updateBody",
+            "required": true,
+            "description": "The update body",
+            "schema": {
+              "$ref": "#/definitions/updateObject"
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Request Accepted Response.",
+            "schema": {
+              "$ref": "#/definitions/acceptedResponse"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/workspace.json#/definitions/ErrorContract"
+            }
+          }
+        }
+      }
+    },
+    "/symssync/delete": {
+      "put": {
+        "tags": [
+          "symssync"
+        ],
+        "operationId": "SymsSync_Delete",
+        "description": "Register files in Syms",
+        "x-ms-examples": {
+          "Register files in Syms": {
+            "$ref": "./examples/SymsSync_Delete.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "body",
+            "name": "deleteBody",
+            "required": true,
+            "description": "The delete body",
+            "schema": {
+              "$ref": "#/definitions/deleteObject"
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Request Accepted Response.",
+            "schema": {
+              "$ref": "#/definitions/acceptedResponse"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/workspace.json#/definitions/ErrorContract"
```

Comment:
>../../../../resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/workspace.json#/definitions/ErrorContract [](http://example.com/codeflow?start=23&length=115)

We should avoid having reference between resource management and data plane APIs.

To do so, can we copy the definition of "ErrorContract" into this file? Reference access control: [https://github.com/Azure/azure-rest-api-specs/blob/master/specification/synapse/data-plane/Microsoft.Synapse/preview/2020-08-01-preview/roleAssignments.json#L213-L265](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/synapse/data-plane/Microsoft.Synapse/preview/2020-08-01-preview/roleAssignments.json#L213-L265)